schema/defs-config.json

```diff
@@ -38,7 +38,10 @@
           }
         },
         "Volumes": {
-          "$ref": "defs.json#/definitions/mapStringObject"
+          "oneOf": [
+            {"$ref": "defs.json#/definitions/mapStringObject"},
+            {"type": "null"}
```

Comment:
This doesn't look like `jq` ;).  Did you run [`make fmt`][1] (with the fix from #148)?

[1]: https://github.com/opencontainers/image-spec/pull/148/files#diff-2cb561c5e46512465f54a0f84d421c80R59